src/Input/input.style.js

```diff
@@ -22,9 +23,16 @@ const InputStyles = styled.div`
     font-size: 18px;
     font-weight: normal;
     color: #353f41;
-    padding: 2px 16px;
     transition: border-color 0.2s ease, box-shadow 0.2s ease;
 
+    &.default-padding {
+      padding: 2px 16px;
+    }
+
+    &.extra-padding {
```

Comment:
Instead of extra why not icon-padding? That way we know what it is for.

Reply:
Also is the & required?

Reply:
& is required yes.

Reply:
its because it is like `input.extra-padding`